caravel/views.py

```diff
@@ -1304,10 +1324,12 @@ def sql_json(self):
                 sql = sql.strip().strip(';')
                 qry = (
                     select('*')
-                    .select_from(TextAsFrom(text(sql), ['*']).alias('inner_qry'))
+                    .select_from(TextAsFrom(text(sql), ['*'])
+                                 .alias('inner_qry'))
```

Comment:
pycharm puts it there. It uses pep8 by default. .alias belongs to the TextAsFrom(text(sql), ['*']), not to the result of the select_from.